src/App.js

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
-import './App.css';
+import { useSelector } from 'react-redux';
 import { Routes, Route } from 'react-router-dom';
+import SplashScreen from './components/SplashScreen/SplashScreen';
 import Register from './components/Register/Register';
 import Login from './components/Login/Login';
+import Doctors from './components/Doctors/Doctors';
+import BookAppointment from './components/Appointments/BookAppointment';
+import MyAppointments from './components/Appointments/MyAppointments';
+import Navbar from './components/layouts/Navbar';
+import AddDoctor from './components/Doctors/AddDoctor';
+import DeleteDoctor from './components/Doctors/DeleteDoctor';
+import DoctorDetails from './components/Doctors/DoctorDetails';
 
 function App() {
```

Comment:
- [ ] OPTIONAL: The images are still not loading: 

![image](https://user-images.githubusercontent.com/57421823/207300693-fec99696-bb03-4862-a402-8ef1a672099a.png)

I recommend you fix them so your project looks more beautiful. 💪